src/detect-engine-analyzer.c

```diff
@@ -924,6 +925,13 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_ICMP_ID: {
+                const DetectIcmpIdData *cd = (const DetectIcmpIdData *)smd->ctx;
+                jb_open_object(js, "id");
+                jb_set_uint(js, "number", cd->id);
```

Comment:
Thoughts on whether this should be called `number`?

Reply:
It's consistent with ack and seq keywords:
```c
              case DETECT_ACK: {
                  const DetectAckData *cd = (const DetectAckData *)smd->ctx;

                  jb_open_object(js, "ack");
                  jb_set_uint(js, "number", cd->ack);
                  jb_close(js); 
                  break; 
              }
              case DETECT_SEQ: {
                  const DetectSeqData *cd = (const DetectSeqData *)smd->ctx;
                  jb_open_object(js, "seq");
                  jb_set_uint(js, "number", cd->seq);
                  jb_close(js);
                  break;
              }
```

Reply:
yeah saw that. Thought that was because they were acknowledgement number and sequence number.. ok then not holding the PR for this. Thank you! :)